src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+  margin: 10px 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  justify-content: flex-start;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  font-size: 12px;
+  margin-left: 20px;
```

Comment:
Be consistent with your *vertical* margins. Add only top or only bottom margin, but not both. Vertical margins can be collapsed in some cases. Here, you have set the left margin for `.nav__link`. If you need to set vertical margins, it's better to stick to either top or bottom, but not both.